Gruntfile.js

```diff
@@ -252,20 +252,44 @@ module.exports = function (grunt) {
                     outfile: 'tmp/browser/test-runner-post-processor.html'
                 }
             },
-            postProcessorPLugin: {
-                src: ['test/browser/less/postProcessorPlugin/*.less'],
+            postProcessorPlugin: {
+                src: ['test/less/postProcessorPlugin/*.less'],
                 options: {
-                    helpers: 'test/browser/runner-postProcessorPlugin-options.js',
+                    helpers: ['test/plugins/postprocess/index.js','test/browser/runner-postProcessorPlugin-options.js'],
```

Comment:
looks great thanks - my only q is - the postprocess test plugin is in its own file and folder.. but the other plugins are in the options files? I think it would be tidier to go for one approach or the other. Don't mind which.